NDB.Covid19/NDB.Covid19.Droid/Views/InitializerActivity.cs

```diff
@@ -108,7 +108,9 @@ private void Continue()
         {
             if (PlayServicesVersionUtils.PlayServicesVersionNumberIsLargeEnough(PackageManager))
             {
-                NavigationHelper.GoToOnBoarding(this, true);
+                //NavigationHelper.GoToOnBoarding(this, true);
+                //NavigationHelper.GoToFarewellSmittestopPage(this);
+                NavigationHelper.GoToSmittestopNotActivePage(this);
```

Comment:
Not needed to change this, as we will change launch activity.

Reply:
It was just for testing, forgot to remove it